packages/core/src/helpers/transaction.ts

```diff
@@ -15,3 +16,10 @@ export function shortenIfTransactionHash(transactionHash: string | Falsy): strin
   }
   return ''
 }
+
+export function isTransactionSlow(transaction: StoredTransaction, slowTransactionThreshold: number): boolean {
```

Comment:
Idea:
```ts
const slow = useTransactionSlow(transactionHash: string) 
```

That way we can:
- check if transaction is slow regardless if it is a stored transaction or just transaction
- use useConfig inside isTransactionSlow

Reply:
Or it could check both Typed Transaction or hash of such transactions through :

`transaction: StoredTransaction | string`

When its a string, expect it to be a hash and when its StoredTransaction, check as usual...